docs/css/style.css

```diff
@@ -0,0 +1,894 @@
+@font-face {
```

Comment:
Until we get the CDN set up we'll need this for it to be used by the gh-pages docs.